File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("credit-card-validation_sdk")

describe("CreditCardValidationSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# CreditCardValidation SDK exists test

require "minitest/autorun"
require_relative "../CreditCardValidation_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = CreditCardValidationSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
